internal/image_test.go

```diff
@@ -23,6 +23,105 @@ func testImage(t *testing.T, context spec.G, it spec.S) {
 		dockerConfig string
 	)
 
+	context("FindLatestImageOnCNBRegistry", func() {
+		it.Before(func() {
+			server = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+
+				switch req.URL.Path {
+				case "/v1/buildpacks/some-ns/some-name":
+					w.WriteHeader(http.StatusOK)
+					fmt.Fprintln(w, `{
+  "latest": {
+    "version": "0.1.0",
+    "namespace": "some-ns",
+    "name": "some-name",
+    "description": "",
+    "homepage": "",
+    "licenses": null,
+    "stacks": [
+			"some-stack"
+    ],
+    "id": "a52bd991-0e17-46c0-a413-229b35943765"
+  },
+  "versions": [
+    {
+      "version": "0.1.0",
+			"_link": "https://registry.buildpacks.io//api/v1/buildpacks/some-ns/some-name/0.1.0"
+    }
+  ]
+}`)
```

Comment:
FYI, JSON is whitespace agnostic. This means you can indent this code so that it lines up a bit nicer with the test cases.

Reply:
I am more than happy to give it a quick run through to indent.

Reply:
Should be all lined up now